src/gpu/lv_gpu.mk

```diff
@@ -9,4 +9,4 @@ VPATH += :$(LVGL_DIR)/$(LVGL_DIR_NAME)/src/gpu
 
 CFLAGS += "-I$(LVGL_DIR)/$(LVGL_DIR_NAME)/src/gpu"
 
-include $(LVGL_DIR)/$(LVGL_DIR_NAME)/src/gpu/sdl/lv_gpu_sdl.mk
\ No newline at end of file
+-include $(LVGL_DIR)/$(LVGL_DIR_NAME)/src/gpu/sdl/lv_gpu_sdl.mk
```

Comment:
I believe this is fixed already in https://github.com/lvgl/lvgl/commit/43729d1502dad0ca797b4b6fb8c69a48c81a2af7, so this change is no longer needed.

Reply:
Got it. I have just removed it.